docs/conf/portland/2018/cfp.rst

```diff
@@ -77,18 +77,18 @@ If you are selected to speak at Write the Docs, we will waive your attendance fe
 
 If your proposal is accepted but you are unable to attend due to travel costs, please let us know and we will do our best to help make it happen.
 
-**You’ll hear from us with our proposal decisions by the end of January 2018.**
+**You’ll hear from us with our proposal decisions by the end of {{cfp.notification}}.**
 
 Questions?
 ----------
 
-If you have any questions, please email us at `portland@writethedocs.org <mailto:portland@writethedocs.org>`_ and let us know.
+If you have any questions, please email us at `{{email}} <mailto:{{email}}>`_ and let us know.
 
 Submit your proposal
 --------------------------
 
 .. raw:: html
 
-	<iframe src="https://docs.google.com/forms/d/e/1FAIpQLSdb-MNRT3TdDTuVIcf-3lZRWDOgguZfjatLr-HLN5ngx5t04w/viewform?embedded=true" width="760" height="850" frameborder="0" marginheight="0" marginwidth="0">Loading...</iframe>
+	<iframe src="{{cfp.url}}?embedded=true" width="760" height="850" frameborder="0" marginheight="0" marginwidth="0">Loading...</iframe>
 
-You can also view `our CFP <https://docs.google.com/forms/d/e/1FAIpQLSdb-MNRT3TdDTuVIcf-3lZRWDOgguZfjatLr-HLN5ngx5t04w/viewform>`_ in its own page.
+You can also view `our CFP <{{cfp.url}}>`_ in its own page.
```

Comment:
Ah, I found this confusing, but what we mean here is that you can look at the CFP form on its own page, yeah? If so, maybe: 
'You can also view `the CFP form <{{cfp.url}}>`_ on its own page.'

Reply:
Gonna be revamped anyway.